MOM_input

```diff
@@ -106,6 +26,28 @@ HFREEZE = 10.0
                                 ! where OBLD is the boundary layer depth. If HFREEZE <= 0 (default),
                                 ! melt potential will not be computed."
 
+C_P = 3992.0
```

Comment:
We'll need to be careful with setting this once we have finalised the equation of state. OM2 uses `CP_OCEAN_PRETEOS10 = 3992.10322329649d0` but that's specific to the pre-TEOS10 EOS. The `ROQUET` value would presumably be the MOM6 default, which matches `CP_OCEAN_TEOS10    = 3991.86795711963d0` in MOM5. We also need to make sure CICE6 uses the same value. See https://github.com/COSIMA/access-om3/issues/53

Reply:
Actually, it looks like CICE6 doesn't use ocean heat capacity.

Reply:
is `ENABLE_THERMODYNAMICS` true?

Reply:
Yes, `ENABLE_THERMODYNAMICS` is true by default

Reply:
I've added a reminder about setting Cp to https://github.com/COSIMA/access-om3/issues/20